samtranslator/model/eventsources/push.py

```diff
@@ -484,6 +484,23 @@ def to_cloudformation(self, **kwargs):
         return resources
 
 
+class AlexaSmartHome(PushEventSource):
```

Comment:
We are lacking pretty hard in the unit tests. I love the functional tests but we can do better.

This file is also getting fairly large. Can we split this out into its own class? What are your thoughts around that?

Reply:
We did this for CloudWatchLogs event and should continue with this approach.